riffdog_aws/resources/elbv2/lb.py

```diff
@@ -18,7 +18,8 @@ class AWSLB(AWSResource):
     resource_type = "aws_lb"
```

Comment:
I want to say this is a good PR to also go through and remove all of these now.

Reply:
Should we also do them to being *args to be future compatible as well?